### ISSUE TITLE
I encountered some problems when executing fulfillOrder

### ISSUE THREAD

**Issue by f-hk** [ProjectOpenSea/seaport-js#132]
I want to use seaport-js to create an order and then use another account to buy nft, here my code
```
const provider = new ethers.providers.Web3Provider(window.ethereum);
const seaport = new Seaport(provider);

const offerer = 0x......(my account 1)
const fulfiller = 0x......(my account 2)

const {executeAllActions} = await seaport.createOrder(
    {
      conduitKey: "0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000",
      offer: [
        {
          itemType: ItemType.ERC1155,
          token: "0xf4910C763eD4e47A585E2D34baA9A4b611aE448C",
          identifier: account1 nft token Id,
        },
      ],
      consideration: [
        {
          amount: ethers.utils.parseEther("0.01").toString(),
          recipient: offerer,
        },
      ],
    },
    offerer
);

const order = await executeAllActions();

console.log("order", order)

const {executeAllActions: executeAllFulfillActions} =
    await seaport.fulfillOrder({
      order,
      accountAddress: fulfiller,
    });

const transaction = executeAllFulfillActions();

console.log("transaction", transaction)

```
Then this error appears in the webpage
![image](https://user-images.githubusercontent.com/107092758/199027823-876e7da4-aafb-4f90-bb26-e1e4540e639a.png)

i m sure that seaport.createOrder is success

How can I solve this problem? Thank you for answering my question！

**Comment by adeebabdulsalam** [comment]
we had this same issue, cost us our pilot project launch being delayed.

it's seaport library issue inside this function "getSummedTokenAndIdentifierAmounts"
it calculates some basis point wrongly and returns the finall payableOverride wrong
we had to manually bypass it to get it working temporirily

**Comment by adeebabdulsalam** [comment]
I believe this is cause by this particular issue  #141

**Comment by luisantoniocrag** [comment]
how can I solve this to make it works?

**Comment by luisantoniocrag** [comment]
> I believe this is caused by this particular issue #141

It seems it is because another reason, because it also fails when try to fill 1 (of max 1 of items) and don't it is handling with decimals.

**Comment by luisantoniocrag** [comment]
Fast solution: https://github.com/luisantoniocrag/seaport-js/commit/fcb3b2e186af6d447d0bdcd02b99017bc81a687c

**Comment by vizsatiz** [comment]
How is this and the other one related ?

**Comment by vizsatiz** [comment]
> Fast solution: [luisantoniocrag@fcb3b2e](https://github.com/luisantoniocrag/seaport-js/commit/fcb3b2e186af6d447d0bdcd02b99017bc81a687c)

How is this solution to the above mentioned approximation problem ?

**Comment by xtools-at** [comment]
@vizsatiz it isn't.

i'm using seaport-js for an enterprise project and I need to get this working, so I'll set out to fix this properly using the demo unit test #145, will create a PR if successful.
if you have any insights you'd like to share to make my life easier, any help's appreciated.

**Comment by xtools-at** [comment]
created PR #197 together with @wooglie to solve this issue, please read the limitations stated in the description
